package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "bs-awesomize-express",
+  "version": "1.0.0",
+  "description": "An ExpressJs integration API for the Awesomizer",
+  "scripts": {
+    "build": "bsb -make-world",
+    "clean": "bsb -clean-world",
+    "install:peers": "yarn add $(jq -r '.peerDependencies|keys|join(\" \")' package.json)",
+    "start": "bsb -make-world -w",
+    "test": "yarn run clean; yarn run build; jest --coverage",
+    "watch:jest": "jest --coverage --watchAll"
+  },
+  "main": "src/AwesomizeExpress.bs.js",
+  "repository": "https://github.com/carmonw/bs-awesomize-express",
+  "author": "Carmon Wong <cwong@influential.co>",
+  "license": "MIT",
+  "peerDependencies": {
+    "bs-platform": "^2.2.3"
+  },
+  "devDependencies": {
+    "@glennsl/bs-jest": "^0.4.2",
+    "@glennsl/bs-json": "^1.2.0",
+    "bs-awesomize": "^1.3.0",
```

Comment:
`bs-awesomize` and `bs-express` should be listed as `dependencies`.

Reply:
As noted in `Things of note`, this will be done Monday.